libraries/azure_service_bus_namespaces.rb

```diff
@@ -0,0 +1,35 @@
+require 'azure_generic_resources'
+
+class AzureServiceBusNamespaces < AzureGenericResources
+  name 'azure_service_bus_namespaces'
+  desc 'Verifies settings for a collection of Azure Service Bus Namespaces in a Resource Group'
+  example <<-EXAMPLE
+    describe azure_service_bus_namespaces(resource_group: 'migrated_vms') do
```

Comment:
```suggestion
    describe azure_service_bus_namespaces(resource_group: 'RESOURCE_GROUP') do
```

Reply:
not following this construct in rb files